Doc/library/asyncio-task.rst

```diff
@@ -282,6 +282,23 @@ Creating Tasks
       Added the *context* parameter.
 
 
+Task Cancellation
+=================
+
+Tasks can easily and safely be cancelled.
+When a task is cancelled, asyncio will raise a :exc:`asyncio.CancelledError`
+into the task at the next opportunity.
```

Comment:
I may just be a n00b, but what exactly does it mean to raise an error _into_ a task? What determines when there will be an "opportunity"?